openbas-api/src/main/java/io/openbas/utils/pagination/SearchPaginationInput.java

```diff
@@ -22,7 +22,7 @@ public class SearchPaginationInput {
   @Min(0) int page = 0;
 
   @Schema(description = "Element number by page")
-  @Max(1000) int size = 10;
+  @Max(1000) int size = 20;
```

Comment:
Why this change ?

Reply:
I found that when I modify an element in the list, and I propagate the change to update the list, it is reloaded but uses the default value set in the backend. And there is an inconsistency between the minimum value of the back which is 10 and that of the front which is 20

Reply:
well seen